packages/label/src/LabelBase.tsx

```diff
@@ -0,0 +1,83 @@
+import { family, weight } from '@littlespoon/theme/lib/fonts/primary'
+import { rem } from '@littlespoon/theme/lib/utils'
+import { allColors } from '@littlespoon/theme/src/colors'
+import styled from 'styled-components'
+
+import type { LabelProps } from './Label'
+
+export const LabelBase = styled.label<LabelProps>`
```

Comment:
Can we change this to a `span` instead of a `label`?

https://github.com/little-spoon-dev/design-system/pull/568#discussion_r940825632

Reply:
So perhaps what is causing confusion is that this component is not a `label` per se but actually a `tag`? I asked [design](https://little-spoon.slack.com/archives/C03DLTCCTUN/p1660076834153319?thread_ts=1660072365.218709&cid=C03DLTCCTUN) if we want to keep the name `Label`

Reply:
Yeah, I think it's the name that the marketing team uses. It has nothing to do with form labels. I think that could be a bit misleading about the intended usage of this component if we keep it with the `label` tag.

Reply:
Used span element